selftest/iter/main.bpf.c

```diff
@@ -4,6 +4,9 @@
 
 #include <bpf/bpf_helpers.h>
 
+const char sleep_comm[] = "sleep";
+const int sleep_comm_size = sizeof(sleep_comm);
```

Comment:
nit: this sizeof can be in the function itself since it is calculated in compile time